content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,232 @@
+
+You can stop tracking a trackable (asset) that is registered with the publisher using the @remove@ method, as shown in the following code:
+
+```[android]
+publisher.remove(trackable)
+```
+
+h2(#subscriber-sdk). Using the Subscriber SDK
+
+Common operations you will need to carry out on the subscriber include:
+
+* Initialization of the subscriber.
+* Listen for updates sent from from the publisher.
+* Listen for asset status updates sent from the publisher.
+* Request a different resolution to be sent from the publisher.
+
+h3(#subscriber-initializing-sdk). Initializing the Subscriber
+
+During initialization of the subscriber various methods can be called to configure the @Subscriber@.
+
+The required methods are:
+
+- connection := Called to provide Ably connection information, such as API keys, and any other configuration parameters as needed. 
+- trackingId := Sets the asset to be tracked, using the unique tracking identifier of the asset.
+
+The optional methods are:
+
+- resolution := Request a specific resolution of updates to be requested from the remote publisher.
+
+The following code example demonstrates initializing and starting the subscriber:
+
+```[android]
+// Initialise and Start the Subscriber
+val subscriber = Subscriber.subscribers() // Get an AssetSubscriber
+    .connection(ConnectionConfiguration(ABLY_API_KEY, CLIENT_ID)) // provide Ably configuration with credentials
+    .resolution( // request a specific resolution to be considered by the publisher
+        Resolution(Accuracy.MAXIMUM, desiredInterval = 1000L, minimumDisplacement = 1.0)
+    )
+    .trackingId(trackingId) // provide the tracking identifier for the asset that needs to be tracked
+    .start() // start listening for updates
+```
+
+h3(#subscriber-subscribe). Subscribe to updates
+
+You can subscribe to updates from the publisher, specifying a function that is called when each update is received. This is shown in the following example:
+
+```[android]
+// Listen for location updates
+locations
+    .onEach { } // provide a function to be called when enhanced location updates are received
+    .launchIn(scope) // coroutines scope on which the locations are received
+```
+
+h3(#subscriber-subscribe-asset-state-changes). Subscribe to asset state changes
+
+You can subscribe to asset state changes from the publisher, specifying a function that is called when each state change is received. This is shown in the following example:
+
+```[android]
+// Listen for asset state changes
+trackableStates
+    .onEach { } // provide a function to be called when the asset changes its state
+    .launchIn(scope) // coroutines scope on which the statuses are received
+```
+
+h3(#subscriber-subscribe-asset-state-changes). Request a different resolution
+
+The subscriber can always request a different resolution preference by calling the @resolutionPreference@ method, passing in the required @Resolution@. This is shown in the following example:
```

Comment:
I think it is important to explain somewhere why we have this feature. The use-case is simple - subscriber may require different resolutions of updates depending on state - i.e. wether map is displayed to the user, what zoom level is set on the map, etc. This feature allows subscriber to express different preferences depending on all these factors. 

Maybe this can be described on the index page?

Reply:
I've added a new section in the overview. 

> "depending on all these factors."

Do we have a complete list of these factors anywhere?